src/js/game.js

```diff
@@ -1,128 +1,156 @@
+            this.accountValue.textContent = `${temporaryAccountValue + moneyWon}$`;
             this.accountMoney = parseInt(this.accountValue.textContent);
-            this.bidValue.value = '';
-            this.bidValue.placeholder = 'Bid and push button'
-            
-            this.stats.collectHistroy(games, wins, loses, bid, bonus, moneyWon, 0, this.accountMoney);
-            console.log(this.stats.gameHistory);
-            
-            this.stats.showStats(this.statsHistory, games, wins, loses, bid, bonus, moneyWon, 0, this.accountMoney);
+
+            this.renderHistoryCard()
+            this.stats.historyCards[this.stats.countCards].appendChild(this.stats.showStats(bid, bonus, moneyWon, 0, this.accountMoney));
+            this.ckeckCardHeight();
             
             if(bonus){
                 this.score.textContent = `+ ${moneyWon} $`;
+                this.score.style.color = 'green';
                 this.score.classList.add('score--show');
             } else{
-                this.score.textContent = `- ${bid} $`
+                this.score.textContent = `- ${bid} $`;
+                this.score.style.color = 'red';
                 this.score.classList.add('score--show');
             }
             setTimeout(() => {
                 this.score.classList.remove('score--show');
                 this.throttle = false;
             },1000);
 
-        } else if(this.throttle){
-            this.bidValue.value = '';
-            this.bidValue.placeholder = 'You play to fast !'
-        }else{
-            this.bidValue.value = '';
+            this.bidValue.placeholder = 'Bid and push button';
+            
+        }else if(this.throttle){
+            this.bidValue.placeholder = 'You bid to fast !';
+        }else {
             this.bidValue.placeholder = 'You need to bid !';
         }
+
+        this.bidValue.value = '';
     }
     
-    render(){
+    renderColors(){
         const draw = new Draw();
-        let colors = draw.drawColors(this.cards.length);
+        let colors = draw.drawColors(this.cards);
         this.cards.forEach((card, index) => {
             card.style.backgroundColor = colors[index];
         })
         return colors
     }
 
     resetGame(){
-        this.accountValue.textContent = ' 1000$'
-        this.accountMoney = parseInt(this.accountValue.textContent);
-        this.gameCount.textContent = ' 0';
-        this.wins.textContent = ' 0';
-        this.loses.textContent = ' 0';
+        this.accountValue.textContent = '1000$';
+        this.accountMoney = parseInt(this.accountValue.textContent, 10);
+        this.gameCount.textContent = '0';
+        this.wins.textContent = '0';
+        this.loses.textContent = '0';
         this.bidValue.value = '';
         this.bidValue.placeholder = 'Bid and push button';
-        this.statsHistory.textContent = '';
-
+        this.dots.textContent = '';
+        this.stats.gameStats.games = 0;
```

Comment:
Wszystko z `this.stats.gameStats.*` powinno być w statistics.js wywoływane w metodzie "reset".